spec/static_association_spec.rb

```diff
@@ -111,6 +109,18 @@ class DummyClass
         it { should be_nil }
       end
     end
+
+    describe ".where" do
+      it "returns records that exist" do
+        record_1 = DummyClass.record(id: 1) { self.name = "First" }
+        record_2 = DummyClass.record(id: 2) { self.name = "Second" }
+        record_3 = DummyClass.record(id: 3) { self.name = "Third" }
+
+        results = DummyClass.where([1, 3])
+
+        expect(results).to eq([record_1, record_3])
```

Comment:
Using `eq` enforces a sorting that might not exist. What do you think of using `contain_exactly` instead?

```suggestion
        expect(results).to contain_exactly([record_1, record_3])
```